torchvision/datasets/utils.py

```diff
@@ -175,9 +175,8 @@ def list_files(root: str, suffix: str, prefix: bool = False) -> List[str]:
 
 
 def _quota_exceeded(response: "requests.models.Response") -> bool:  # type: ignore[name-defined]
-    return False
-    # See https://github.com/pytorch/vision/issues/2992 for details
-    # return "Google Drive - Quota exceeded" in response.text
+    start = next(response.iter_content(chunk_size=128, decode_unicode=True))
```

Comment:
Is there a risk that this raises a StopIteration in some scenarios?

Reply:
IIRC no, `iter_content` should handle shorter chunks automatically. If you want to be sure I can wrap it,

Reply:
yeah, in doubt, let's protect it